rules/windows/defense_evasion_whitespace_padding_in_command_line.toml

```diff
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2021/07/30"
+maturity = "production"
+updated_date = "2021/07/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution events where the command line value contains superfluous whitespace characters. Attackers
+may attempt to evade signature-based detections by padding their malicious command with unnecessary whitespace
```

Comment:
you could also note that what the spaces indicate visually. how it causes the "bad part" to overflow off the screen or text box